compiler/src/iree/compiler/Codegen/Dialect/CPU/IR/IREECPUAttrs.cpp

```diff
@@ -232,11 +233,11 @@ constexpr std::array vectorTilingLevels{TilingLevel::VectorCommonParallelTiles,
 std::optional<SmallVector<int64_t>> LoweringConfigAttr::getVectorSizes() const {
   SmallVector<int64_t> result;
   for (auto level : vectorTilingLevels) {
-    if (!hasTilingLevel(level)) {
+    if (!hasTilingLevel(static_cast<unsigned>(level))) {
```

Comment:
It might be good to update these methods to take in int instead of unsigned in a follow up.